mono/utils/dlmalloc.c

```diff
@@ -461,7 +461,9 @@ DEFAULT_MMAP_THRESHOLD       default: 256K
 #endif  /* _WIN32 */
 #endif  /* WIN32 */
 #ifdef WIN32
+#ifndef WIN32_LEAN_AND_MEAN
```

Comment:
If you define it to be 1 instead of empty you should be ok.

Reply:
@jaykrell Should we make a change here, or is this code fine as-is?

Reply:
Ok asis.